src/code.cloudfoundry.org/gorouter/integration/route_services_test.go

```diff
@@ -69,6 +92,30 @@ var _ = Describe("Route services", func() {
 					Expect(err).ToNot(HaveOccurred())
 				}))
 
+		wsRouteService = httptest.NewUnstartedServer(
+			&httputil.ReverseProxy{
+				Director: func(req *http.Request) {
+					forwardedURLStr := req.Header.Get("X-Cf-Forwarded-Url")
+
+					forwardedURL, err := url.Parse(forwardedURLStr)
+					if err != nil {
+						log.Printf("ERROR: X-Cf-Forwarded-Url unparseable: %s\n", err.Error())
+						return
+					}
+
+					req.URL = &url.URL{
+						Scheme: "http",
+						Host:   fmt.Sprintf("127.0.0.1:%d", testState.cfg.Port),
+					}
+					req.Host = forwardedURL.Host
+				},
+				Transport: &http.Transport{
+					TLSClientConfig: &tls.Config{
+						MinVersion:         tls.VersionTLS12,
+						InsecureSkipVerify: true, //nolint:gosec // required, as route-service needs to talk to crt-test, which does not have a valid certificate
```

Comment:
I think this comment got copied from somewhere else, please remove it. If it fails because of TLS errors we will have to check where to get the correct TLS config from.